### PR TITLE
ADEN-8031 Adding MEGA to MR

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

**Review by fraszczakszymon — APPROVED** [review]

File: extensions/wikia/AdEngine/js/context/adContext.js [hunk 1]
```diff
@@ -101,7 +101,9 @@ define('ext.wikia.adEngine.adContext', [
 		var hasFeaturedVideo = context.targeting.hasFeaturedVideo;
 
 		context.bidders.prebid = !areDelayServicesBlocked() && isEnabled('wgAdDriverPrebidBidderCountries');
-		context.bidders.prebidAE3 = context.targeting.skin === 'oasis' && isEnabled('wgAdDriverPrebidAdEngine3Countries');
+		// Due to TOP_BOXAD rename we have to force new Prebid - otherwise it won't bid for this slot
+		// context.bidders.prebidAE3 = context.targeting.skin === 'oasis' && isEnabled('wgAdDriverPrebidAdEngine3Countries');
+		context.bidders.prebidAE3 = true;
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/context/adContext.js]: what about mobile-wiki?

File: skins/oasis/css/core/ads.scss [hunk 2]
```diff
@@ -203,7 +203,7 @@ $global-navigation-and-sitewide-messages-index: 5000101;
 
 /* grid layout transition temp code.  remove and integrate into main code after transition is complete */
 .WikiaGrid {
-	#TOP_RIGHT_BOXAD {
+	#TOP_BOXAD {
```
Comment (fraszczakszymon) [inline skins/oasis/css/core/ads.scss]: Cache is going to hurt us 🙈

File: skins/oasis/css/core/ads.scss [hunk 3]
```diff
@@ -217,7 +217,7 @@ $global-navigation-and-sitewide-messages-index: 5000101;
 	}
 }
 
-#TOP_RIGHT_BOXAD {
+#TOP_BOXAD {
```
Comment (fraszczakszymon) [inline skins/oasis/css/core/ads.scss]: Cache is going to hurt us 🙈

File: skins/oasis/modules/BodyController.class.php [hunk 4]
```diff
@@ -161,7 +161,7 @@ public function getRailModuleList() {
 			$railModuleList = [
 				1202 => [ 'Forum', 'forumRelatedThreads', null ],
 				1201 => [ 'Forum', 'forumActivityModule', null ],
-				1490 => [ 'Ad', 'Index', [ 'slotName' => 'TOP_RIGHT_BOXAD' ] ],
+				1490 => [ 'Ad', 'Index', [ 'slotName' => 'TOP_BOXAD' ] ],
```
Comment (fraszczakszymon) [inline skins/oasis/modules/BodyController.class.php]: Cache is going to hurt us 🙈

File: skins/oasis/modules/BodyController.class.php [hunk 5]
```diff
@@ -239,7 +239,7 @@ public function getRailModuleList() {
 			return [ ];
 		}
 
-		$railModuleList[1440] = [ 'Ad', 'Index', [ 'slotName' => 'TOP_RIGHT_BOXAD' ] ];
+		$railModuleList[1440] = [ 'Ad', 'Index', [ 'slotName' => 'TOP_BOXAD' ] ];
```
Comment (fraszczakszymon) [inline skins/oasis/modules/BodyController.class.php]: Cache is going to hurt us 🙈

File: extensions/wikia/AdEngine/js/context/adContext.js (hunk 1 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/context/adContext.js]: On mobile-wiki we use AE3 so it's not an issue I guess?

File: skins/oasis/css/core/ads.scss (hunk 2 above)
Comment (dmnsobczak) [reply skins/oasis/css/core/ads.scss]: Yeah, it will :/ Do you have an idea how to workaround this?

Comment (fraszczakszymon) [reply skins/oasis/css/core/ads.scss]: Double styles, for a while

File: skins/oasis/css/core/ads.scss (hunk 3 above)
Comment (fraszczakszymon) [reply skins/oasis/css/core/ads.scss]: Double styles, for a while

File: skins/oasis/modules/BodyController.class.php (hunk 4 above)
Comment (fraszczakszymon) [reply skins/oasis/modules/BodyController.class.php]: Inject both slots, for a while. It shouldn't hurt us. I believe

File: skins/oasis/modules/BodyController.class.php (hunk 5 above)
Comment (fraszczakszymon) [reply skins/oasis/modules/BodyController.class.php]: Inject both slots, for a while. It shouldn't hurt us. I believe

File: skins/oasis/css/core/ads.scss (hunk 2 above)
Comment (dmnsobczak) [reply skins/oasis/css/core/ads.scss]: Fixed.

File: skins/oasis/css/core/ads.scss (hunk 3 above)
Comment (dmnsobczak) [reply skins/oasis/css/core/ads.scss]: Fixed.

File: skins/oasis/modules/BodyController.class.php (hunk 4 above)
Comment (dmnsobczak) [reply skins/oasis/modules/BodyController.class.php]: Fixed.

File: skins/oasis/modules/BodyController.class.php (hunk 5 above)
Comment (dmnsobczak) [reply skins/oasis/modules/BodyController.class.php]: Fixed.

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**